amd/src/response_panel.js

```diff
@@ -227,6 +227,11 @@ function($, ModalFactory, ModalResponse, ModalEvents) {
         });
         docElement.on('click', 'button.proctoringimage', function(e) {
             e.preventDefault();
+            if (release && parseFloat(release) < 4.3) {
+                var type = ModalResponse.TYPE;
+            } else {
+                var type = ModalProctoringImages.TYPE;
+            }
```

Comment:
Change this code to 
            var type = ModalProctoringImages.TYPE;
            if (release && parseFloat(release) < 4.3) {
               type = ModalResponse.TYPE;
            }